### PR TITLE
Feature/login form

### PR DESCRIPTION
**Pull request by Datikzz**

### CODE REVIEW

File: pages/login.vue [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+      <v-icon>mdi-arrow-left-circle-outline</v-icon>
+    </v-btn>
+    <div class="login-form">
+      <v-tabs v-model="currentTab">
+        <v-tab :ripple="false" v-for="tab in tabs" :key="tab">{{ tab }}</v-tab>
+      </v-tabs>
+
+      <v-tabs-items v-model="currentTab">
+        <v-tab-item>
+          <form class="login-form" @submit.prevent="submitHandler">
```
Comment (STsaruk) [inline pages/login.vue:21]: we have the same form repeated three times. 

I have two suggestions: 
1) Use one form for all tabs and use the name of the tab as the identifier of where the form comes from
2) Make a form component and display it in each tabs

I like the first option better. What do you think?

File: pages/login.vue [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+                required
+                placeholder="Password"
+                class="login-row py-3 px-5 w-full bg-white rounded-lg"
+                v-model="form.password"
+              />
+              <a href="#" class="login-row forgot-password-btn pl-[18px] link"
+                >Forgot password?</a
+              >
+              <div class="login-row flex flex-col">
+                <a
```
Comment (STsaruk) [inline pages/login.vue:41]: The link-button also asks for a component

File: pages/login.vue [hunk 3]
```diff
@@ -0,0 +1,296 @@
...
+                    bg-white
+                    rounded-lg
+                    text-center
+                  "
+                  >Sign in with Facebook</a
+                >
+              </div>
+              <p class="login-row text-center mb-0">
+                New to Map of Interaction?
+                <a href="#" class="join-btn link">Join now</a>
```
Comment (STsaruk) [inline pages/login.vue:73]: I think it should be an internal link . Use <NuxtLink to="/">Join now</NuxtLink> for that

File: pages/login.vue [hunk 4]
```diff
@@ -0,0 +1,296 @@
+<template>
+  <div class="login-page">
+    <v-btn
+      rounded
+      fab
+      small
+      icon
+      :ripple="false"
+      class="nav-back-btn"
+      @click="to"
```
Comment (STsaruk) [inline pages/login.vue]: `to` is a bad name for a method, there must be some action `goToBack` for example

File: pages/login.vue (hunk 2 above)
Comment (STsaruk) [reply pages/login.vue:41]: Along with styles

**Review by STsaruk — COMMENTED** [review]
All in all, it looks great!  There are some comments, but If you don't have time for it now, we can fix it later. Great job!

File: pages/login.vue (hunk 4 above)
Comment (chudaol) [reply pages/login.vue]: agree

File: pages/login.vue (hunk 1 above)
Comment (chudaol) [reply pages/login.vue:21]: I agree with the first option. even if we have some different fields in the future, we will be able to add them according to the tab we are

File: pages/login.vue (hunk 3 above)
Comment (Datikzz) [reply pages/login.vue:73]: i will leave like this for now

File: pages/login.vue (hunk 2 above)
Comment (Datikzz) [reply pages/login.vue:41]: fixed formatting,  we'll need to adjust project's linter/formatter

File: pages/login.vue (hunk 4 above)
Comment (Datikzz) [reply pages/login.vue]: my bad, didnt cleanup properly

**Comment by Datikzz** [comment]
Resolved conversations, we will modify form-tab logic later on, stay tuned

File: pages/login.vue (hunk 1 above)
Comment (Datikzz) [reply pages/login.vue:21]: first option is great if we keep same layout and same input order, lets see on next steps how we can improve this

**Pull request merged**